Kickstarter-iOS/Views/Controllers/LoginViewController.swift

```diff
@@ -166,6 +174,14 @@ internal final class LoginViewController: UIViewController {
     self.navigationController?.pushViewController(vc, animated: true)
   }
 
+  fileprivate func startEmailVerificationViewController() {
```

Comment:
I declared this method in both view controllers and want to find a way to only declare this in a single place. I wasn't sure if there was a shared file I could put this? I thought about `SharedFunctions`?